openwisp_radius/base/models.py

```diff
@@ -557,6 +557,20 @@ def close_stale_sessions(cls, days=None, hours=None):
             session.terminate_cause = "Session Timeout"
             session.save()
 
+    @classmethod
+    def close_stale_sessions_for_nas(cls, nas_ip, organization_id):
+        if not nas_ip or not organization_id:
+            return 0
+        stale_sessions = cls.objects.filter(
+            nas_ip_address=nas_ip,
+            organization_id=organization_id,
+            stop_time__isnull=True,
+        )
+        closed_count = stale_sessions.update(
+            stop_time=now(), terminate_cause="Stale-Session"
```

Comment:
I think we can use:

```suggestion
            stop_time=now(), terminate_cause="NAS Reboot"
```

For more info refer to the definition of `Acct-Terminate-Cause` in the RADIUS accounting RFC.